tests/sycl/accessor.cpp

```diff
@@ -161,6 +161,17 @@ BOOST_AUTO_TEST_CASE(accessor_api) {
     cgh.single_task<class accessor_api_device_accessors>([](){});
   });
 
+  queue.submit([&](s::handler& cgh) {
+    run_test([&](auto buf, auto... args) {
+      return buf.template get_access<s::access::mode::atomic>(cgh, args...);
+    });
+    // mostly compilation test
+    auto atomicAcc = buf_a.template get_access<s::access::mode::atomic>(cgh);
+    cgh.single_task<class accessor_api_atomic_device_accessors>([atomicAcc](){
+      atomicAcc[0].exchange(0);
+    });
+  });
```

Comment:
Philosophical question: Do we actually need to/Should we test deprecated features?

Reply:
I think as long as we did not remove them, we should test it, as we still provide it and the user will expect it to work as long as it's there.